spec/models/user_spec.rb

```diff
@@ -1,92 +1,162 @@
   end
 
-  context "#check_industry" do
-  	it "should be true if no current goal" do
-  		user = User.create!(email: "hellowerd2@gmail.com", password: "Somethingwierd12",password_confirmation: "Somethingwierd12", first_name: "Hello",last_name: "world", mentor: true, primary_industry: "Technology", stage_of_career: 5, mentor_industry: "Technology", peer_industry: ["Business", "Technology", "Startup"].sample, current_goal: nil,top_3_interests: ["Arts", "Music", "Crafting", "Home improvement / Decorating", "Being a mom", "Dogs", "Cats", "Watching Sports", "Outdoors / Hiking", "Exercise", "Biking", "Yoga", "Running", "Beer","Wine","Traveling"," Local events",    "Reading", "Photography", "Movies","Cooking / Eating / Being a foodie" ,"Social issues / volunteering","Video Games"].sample(3), live_in_detroit: %w(true false).sample, is_participating_next_month: true, is_participating_this_month: false,
- 				mentor_times: 0, mentor_limit: 1, is_assigned_peer_group: true)
-  		expect(user.waitlist).to eq(true)
-  	end
-
-  	it "should be false if everything is put in right" do
-  		user = User.create!(email: "hellowerd2@gmail.com", password: "Somethingwierd12",password_confirmation: "Somethingwierd12", first_name: "Hello",last_name: "world", mentor: true, primary_industry: "Technology", stage_of_career: 5, mentor_industry: "Technology", peer_industry: ["Business", "Technology", "Startup"].sample, current_goal: ["Rising the ranks / breaking the glass ceiling","Switching industries","Finding work/life balance"].sample,top_3_interests: ["Arts", "Music", "Crafting", "Home improvement / Decorating", "Being a mom", "Dogs", "Cats", "Watching Sports", "Outdoors / Hiking", "Exercise", "Biking", "Yoga", "Running", "Beer","Wine","Traveling"," Local events",    "Reading", "Photography", "Movies","Cooking / Eating / Being a foodie" ,"Social issues / volunteering","Video Games"].sample(3), live_in_detroit: %w(true false).sample, is_participating_next_month: true, is_participating_this_month: false,
- 				mentor_times: 0, mentor_limit: 1, is_assigned_peer_group: true)
-  		expect(user.waitlist).to eq(false)
-  	end
-
-  	it "should be true if primary industry is nil" do
-  		user = User.where(waitlist: false).sample
-  		user.update(primary_industry:nil)
-  		expect(user.waitlist).to eq(true)
-  	end
-
-  	it "should be true if primary industry is Other" do
-  		user = User.where(waitlist: false).sample
-  		user.update(primary_industry:"Other")
-  		expect(user.waitlist).to eq(true)
-  	end
-
-  	it "should be true if top_3_interests is []" do
-  		user = User.where(waitlist: false).sample
-  		user.update(top_3_interests: [])
-  		expect(user.waitlist).to eq(true)
-  	end
+  describe '.update_month' do
+    #update_month 'refreshes' user settings for next month
+
+    context 'if user is participating this month' do
+      let!(:participating_user) do
+        create(:skinny_user,
+                           is_participating_this_month: true,
```

Comment:
Thanks for catching. I just switched text editors, so I think me getting used to new shortcuts resulted in a few errant spaces. No need to link indentation styleguide, but thanks. :)